### ISSUE TITLE
Native tooltip eating up abnormal GPU usage on transparent window

### ISSUE THREAD

**Issue by ButzYung** [electron/electron#5971]
* Electron version: 1.2.2 (32-bit)
* Operating system: Windows 10 Home (64-bit)

The native tooltip that is displayed on any HTML element with a "title" attribute, is sometimes causing significant GPU usage on a transparent Electron window. The problem is demonstrated in the following video.

https://www.youtube.com/watch?v=nmrKkt8Xefg

The Electron demo appeared to be normal at first with normal GPU usage, but when I started to move around the cursor to toy with the tooltip on the lower left corner (0'52"), the display of tooltip would mess up at certain point, and the GPU load and memory controller load began to skyrocket, which not only slowed its own display but also locked up the display of other windows. The problem persisted until the window was minimized or closed (or when the tooltip window naturally disppeared).

The problem is still there even if I replace it with a much simpler animation. It doesn't seem to have much relationship with the nature of the content. As long as the window is large enough and there are some tooltips to display, the problem will show up sooner or later.

The problem does NOT seem to appear on a NON-transparent window.

Not totally related, but the problem doesn't occur when I run it on NW.js.

PS: I am seriously considering moving my current desktop gadget project from NW.js to Electron, mainly due to the new "click-through" window feature introduced in Electron v1.2.2. This tooltip problem is bothering me somehow, but I have a workaround by replacing the native tooltip with an inline CSS version.

**Comment by zcbenz** [comment]
What's the version of Chrome in NW.js you used? And did you disable hardware acceleration in it?

**Comment by ButzYung** [comment]
@zcbenz  The current NW.js version I am using for my app is v0.12.3, which should be based on Chromium 41.0.2272.76, and yeah it's pretty old already (I haven't upgraded to v0.13+ yet due to some technical issues, another reason why I am considering moving to Electron).

Hardware acceleration is always on (my app uses a lot of canvas and WebGL stuff, so it has to be on anyways).

I tested my app on older versions of Electron. The problem was still there on v1.2.1 and v1.1.3. I didn't test all versions, and the last one I tested was v1.0.1. Suprisingly, v1.0.1 worked fine with tooltip displayed properly without GPU lockup. I guess it may have something to do with the Chromium version?

As mentioned in the first post, the problem is gone if BrowserWindow is non-transparent.

Not sure if it's related, but newer versions of Electron crashed my app sometimes (ie. everything on the Electron window disappeared after app startup, though this totally transparent window was still occupying screen space with mouse blocking etc), especially with IFRAME and some WebGL animations. This NEVER happened before on NW.js, and so far not on v1.0.1 as well.

**Comment by zcbenz** [comment]
I think you have encountered a regression of Chromium. It works in old versions of NW.js and Electron because they are using old versions of Chromium.

This is essentially a bug of Chromium, and I suggest reporting it to Chromium's issues list instead, I'm closing this because we are unable to fix it on Electron's side.

**Comment by ButzYung** [comment]
@zcbenz I have filed the bug report to Chromium's side, but anyways after some testings, I have managed to figure out the simplest steps to reproduce the problem. I suppose these information may be useful to you.

1) Open an Electron (Chromium 50+) window in transparent mode.

2) Open a page with some elements that contain the "title" attribute.

3) Resize the window after page load (using something like window.resizeBy(640,480)).

4) Move the cursor around the page so that those tooltips are shown. At certain point, the GPU usage will suddenly rise noticeably.

Attached is a simple app that can reproduce the problem.
[app.zip](https://github.com/electron/electron/files/321000/app.zip)
